gdcdatamodel/models/__init__.py

```diff
@@ -27,6 +27,7 @@
 import notifications
 import submission
 import redaction
+import download_reports
```

Comment:
We might need to move the report models into a different package so that when something does `from gdcdatamodel.models import *`, it doesn't assume the report tables should go into the graph DB.

Reply:
yeah @anmaxvl and @kulgan noted this issue as well -- I'll DM you after our meetings today to discuss.

Reply:
moving to another repo `gdc_ng_models`